iri/testing/machine/fake.go

```diff
@@ -70,6 +71,32 @@ type FakeRuntimeService struct {
 	GetExecURL         func(req *iri.ExecRequest) string
 }
 
+// ListEvents implements machine.RuntimeService.
+func (r *FakeRuntimeService) ListEvents(ctx context.Context, req *iri.ListEventsRequest) (*iri.ListEventsResponse, error) {
```

Comment:
Can you please provide an option to set events manually like with `SetEvents`?

Reply:
Done added SetEvents()